keadm/app/cmd/util/edgecoreinstaller.go

```diff
@@ -51,9 +51,12 @@ func (ku *KubeEdgeInstTool) InstallTools() error {
 	ku.SetOSInterface(GetOSInterface())
 	ku.SetKubeEdgeVersion(ku.ToolVersion)
 
-	err := ku.InstallKubeEdge()
-	if err != nil {
-		return err
+	var err error
+	if InstallersOnline {
+		err := ku.InstallKubeEdge()
+		if err != nil {
+			return err
+		}
```

Comment:
In cloudcore, If offline mode, we only start binary, don't copy again, why not add https://github.com/kubeedge/kubeedge/pull/1266/files#diff-f7a2c5d5ad972d622b21fe88054b53c3R184-R193 here?

Reply:
thx maybe I'll share an other patch

Is the feature still desirable ?

/assign @kevin-wangzefeng